string obj_name
---
bool confirmation
string location
float64 probability
geometry_msgs/Pose object_pose
